### ISSUE TITLE
Display Keyboard Shortcuts in Videos

### ISSUE THREAD

**Issue by bayer** [mage2tv/issues#8]
As a proficient PHPStorm user, you seem to be using many keyboard shortcuts. 

It would definitely help if you would display those shortcuts in the videos, so the person watching could more easily follow your steps & learn to use those shortcuts.

**Comment by Vinai** [comment]
Thanks for your issue report, Daniel!
You raise a valid point. We have thought about it quite a bit.
The software we chose to make the recordings - ScreenFlow - supports displaying the keyboard combinations on the recordings. We turned the display on originally, but wheren't satisfied with the result. It turned out to be quite confusing.
Ideally the viewer would be able to turn the display of hortcuts on and off herself, but we currently have no way of realizing that.

Showing the shortcuts also is problematic if the viewer uses Windows or Linux, since the screen recording is done on MacOS.
Additionally, even though I mainly use the defaults, I do have configured a few custom shortcuts.

Another observation of mine is that when I see many keyboard shortcuts in a video, I remember none. Instead I try to learn a new shortcut every week or two, and start using it in day to day work. That works well, at least for me.

So, long story short, I think I'll start doing a "keyboard shortcut of the week" series or something similar, once Mage2 TV is online and transacting, rather then displaying the shortcuts I use in every videos.
If we find a way to display the shortcuts optionally without it being too much overhead during production, then we will do that.
If you have additional thoughts on this, please let me know.
